Signal/src/AppDelegate.m

```diff
@@ -187,6 +193,10 @@ - (void)applicationWillTerminate:(UIApplication *)application
     OWSLogInfo(@"applicationWillTerminate.");
 
     [DDLog flushLog];
+    
+    if (_lokiP2PServer) {
```

Comment:
You only need to use `_lokiP2PServer` (instead of `self.lokiP2PServer)` if you're trying to get around the setter of the property